### PR TITLE
Load signer from the contract

### PR DESCRIPTION
**Pull request by biern**
- [x] PS Contracts API
- [x] Minimal test data
- [x] Populate test data with npm start
- [x] Populate test data for e2e testing
- [x] E2E test
- [x] Clean it up

### CODE REVIEW

**Comment by biern** [comment]
@jstefanski I still need to do some cleanups here, but in case it's blocking your work this is how I've solved the testrpc / seed / contracts deployment / rpc fixtures in e2e environment issues:
https://github.com/SignHash/signhash-webapp/pull/24/commits/515497831850e735ed141db6ef7d757fdab681c0
It seems pretty solid to me.

File: src/Lib/SignHash/Contracts.js [hunk 1]
```diff
@@ -12,17 +12,40 @@ var abis = R.map(requireABI, {
 });
 
 
+function fixTruffleAPI(contract) {
+  // This rebinds `.call` of all contract methods to default
+  // `Function.prototype.call` implementation. Some environments /
+  // packages don't except .call to be rebound to a custom behaviour
+  // and fail unexpectedly when working with `truffle-contract`
+  // (eg. testcafe and it's hammerhead).
```
Comment (jksf) [inline src/Lib/SignHash/Contracts.js]: Typo: it's -> its

File: src/Lib/SignHash/Contracts.js [hunk 2]
```diff
@@ -12,17 +12,40 @@ var abis = R.map(requireABI, {
 });
 
 
+function fixTruffleAPI(contract) {
+  // This rebinds `.call` of all contract methods to default
+  // `Function.prototype.call` implementation. Some environments /
+  // packages don't except .call to be rebound to a custom behaviour
```
Comment (jksf) [inline src/Lib/SignHash/Contracts.js]: Let's stick to American English: behaviour -> behavior

**Comment by jksf** [comment]
I fixed typos, I will merge it as soon as it passes CI (just in case :-)).

Next, I will work on migrating NPM script to use `signhash-contracts` NPM package instead of git and new `seed` script.

**Pull request merged**